src/matlab_src/methAnalysisForChr.m

```diff
@@ -22,7 +22,7 @@
 %%%%%%%%  informME: Information-Theoretic analysis of Methylation  %%%%%%%%
 %%%%%%%%                   methAnalysisForChr.m                    %%%%%%%%
 %%%%%%%%          Code written by: W. Garrett Jenkinson            %%%%%%%%
-%%%%%%%%                Last Modified: 12/08/2016                  %%%%%%%%
+%%%%%%%%                Last Modified: 06/01/2018                  %%%%%%%%
```

Comment:
Can we go through and remove all dates? Github will do a better job tracking than us.

Reply:
Sure thing, I like that.